### ISSUE TITLE
Respect System Font Settings

### ISSUE THREAD

**Issue by OriginalFannibal** [standardnotes/desktop#87]
I'm not sure if Standard Notes is presently shipping its own fonts or using a font stack with hardcoded fonts but it would sure be nice if the desktop app just inherited the system font settings. This would really help users who have changed their system font, a fairly easy and common customization on Linux. It would go some way towards making the app feel more native and also offer a more consistent experience with the rest of the system.

It would also be really nice if the app could also inherit the system monospace font settings as well.

Seeing how this is an Electron app, I think this might be of some help: [15080](https://github.com/atom/atom/pull/15080). That PR is attempting to do the exact same thing I am talking about here to Atom.

**Comment by mobitar** [comment]
It used to be like this. However, for some inexplicable reason, on Mac the Electron app's monospace font was unusable. You could hardly see it. So I added [this rule](https://github.com/standardnotes/desktop/blob/master/app/index.html#L64): `font-family: Menlo, Consolas, 'DejaVu Sans Mono', monospace;`

**Comment by OriginalFannibal** [comment]
What about the default UI font and the proportional font for the editor? Can they please be set to `system-ui`?

**Comment by mobitar** [comment]
`system-ui` wouldn't necessarily be a good editing font. Can you show me how the current font looks for you on your OS?

**Comment by OriginalFannibal** [comment]
> `system-ui` wouldn't necessarily be a good editing font.  

Fair enough. Would it be possible to use it as the UI font while letting users chose the editing font? I feel that the UI font definitely should be consistent with the rest of the system. I see that presently the toolbar does seem to be using my system font (Fira Sans) but the list of notes, list of tags and settings for the editor don't.

Here is a screenshot of what the current editor font looks like:  
![screenshot from 2017-08-19 06-56-24](https://user-images.githubusercontent.com/20235646/29482252-9dbe8556-847d-11e7-90f9-1bef63c98358.png)

Readability isn't really an issue but I would rather have Spectral as my font, the way I do on Atom. Can that setting be exposed?

**Comment by mobitar** [comment]
Currently the way to do this is with themes. I would like to expose this as a simple setting in the future. I've made a note of it, thanks.
